### ISSUE TITLE
Should there be only one `<h1>` tag per page for SEO and a11y?

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#5597]
# Story

As a site owner, I can only have one `<h1>` tag per page for SEO and a11y purposes. 

There is some debate on whether sites should have one or two `<h1>` tags per page: https://github.com/uiowa/uiowa/issues/5597#issuecomment-1232063974. 

Some sites are getting reported for missing a page title https://github.com/uiowa/uiowa/issues/5597#issuecomment-1220696162. 

**Possible ideas:**
- Manually add page titles to layout builder templates that were previously removed https://github.com/uiowa/uiowa/issues/5597#issuecomment-1220696162.

**Comment by bspeare** [comment]
T.M. also asked for the site-name to be an `<h1>` back in https://github.com/uiowa/uiowa/issues/1942#issue-655924007, so that could be the reason we set that as well.

**Comment by joewhitsitt** [comment]
Not sure I am following this and I think I missed where/when this came about.

Tippie and Admissions do have page-titles, they are just visually hidden "Home." Editors have the control over what the page title should be. If there is a missing page-title, something is broken.

<img width="616" alt="Screen Shot 2022-08-18 at 4 54 42 PM" src="https://user-images.githubusercontent.com/4663676/185502042-bcf66199-9255-4e77-9656-07e7ee8c842d.png">

Could someone point me to where WebAIM, WCAG, etc. states this, or the SEO penalty we are receiving because of this setup, because I recall trying to handle the DOM heading structure correctly with regards to HTML5/SEO/A11y early on in D8.

**Comment by bspeare** [comment]
@joewhitsitt I'm still not seeing an `<h1>` on Tippie though.

**Comment by joewhitsitt** [comment]
I guess you are right. Looks like the header was completely removed from this page by a developer. We shouldn't be doing that.

I was looking at the Iowa MBA page that does have it

**Comment by bspeare** [comment]
Sites with home pages with only one `<h1>`:
- https://grad.uiowa.edu/.  
- https://policy.clas.uiowa.edu/
- https://education.uiowa.edu/
- https://nursing.uiowa.edu/
- https://stanleymuseum.uiowa.edu/
- https://tippie.uiowa.edu/
- https://dentistry.uiowa.edu/

**Comment by joewhitsitt** [comment]
Not sure the use-case for why they were removed. Header is supposed to be locked down.

**Comment by bspeare** [comment]
I agree, but just from the short list compiled above we will probably need to check every `node/1` for h1s before we make this change - right?

**Comment by joewhitsitt** [comment]
It might extend beyond node/1s since it appears to be more common than I thought and the possibility of replicate or re-assigning the front page to another node. Have developers done this with any interior pages like this (sub-site look and feel)?

**Comment by bspeare** [comment]
From T M:

> When in doubt, I almost always defer to the good people at WebAIM (webaim.org). They’re the best and have been at it the longest. WebAIM says a document should “typically” have only one H1 heading.

> With that in mind, I see nothing in WCAG or in Open Ajax that limits the number of H1 elements per page for a11y or otherwise. I’m still a bit of a “one H1” acolyte—that’s how I will approach web docs—but I’d position that as a best practice more so than a “standard”. Maybe something like:

> “Be mindful of how you structure your document headings. Some users may navigate your web content in descending outline order. For those users, multiple H1 elements might be confusing.”



> That’s my 0.02 anyway; hope it is worth at least that much.
